prac2-pages/Siddharth_gautam_login_page/lib/main.dart

```diff
@@ -0,0 +1,198 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+
+      home: Scaffold(
+
+        body: const MyStatefulWidget(),
+
+      ),
+    );
+  }
+}
+
+class MyStatefulWidget extends StatefulWidget {
+  const MyStatefulWidget({Key? key}) : super(key: key);
+
+  @override
+  State<MyStatefulWidget> createState() => _MyStatefulWidgetState();
+}
+
+class _MyStatefulWidgetState extends State<MyStatefulWidget> {
+  TextEditingController nameController = TextEditingController();
+  TextEditingController passwordController = TextEditingController();
+  bool value=false;
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
```

Comment:
It is a nice idea to wrap your entire content in a **SafeArea** widget.
Kindly go through the docs to know what it is and how you can use it. :thought_balloon: